replica_aggregator/src/main/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+aws.sqs.queue.url=${?queue_url}
+aws.ingest.sns.topic.arn=${?ingest_topic_arn}
+aws.outgoing.sns.topic.arn=${?outgoing_topic_arn}
+aws.metrics.namespace=${?metrics_namespace}
+operation.name=${?operation_name}
+aws.locking.dynamo.tableName=${?locking_table_name}
+aws.locking.dynamo.tableIndex=${?locking_table_index}
+aws.versions.dynamo.tableName=${?versions_table_name}
+aws.versions.dynamo.tableIndex=${?versions_table_index}
```

Comment:
Very minor, but since this is new: could we alphabetise this list? Just makes it a tad easier to read and see if a particular flag is defined.